### ISSUE TITLE
--quiet argument seems to be ignored by R Host when using MRO

### ISSUE THREAD

**Issue by Hlynsson** [Microsoft/RTVS#1676]
I'm using MRO 3.2.3 and set the command line arguments to inlcude --quiet but it seems to be ignored when the host is started.

Additional information:

OS Information
    Version:       Microsoft Windows NT 10.0.10586.0

RTVS Information:
    Assembly: Microsoft.VisualStudio.R.Package, Version=0.3.20418.1523, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a

R Install path:
    C:\Program Files\Microsoft\MRO\R-3.2.3

**Comment by MikhailArkhipov** [comment]
Can you please provide more information when this is important. Thanks.

**Comment by Hlynsson** [comment]
Not essential but used to suppress startup messages in the interactive session, e.g:
"C:\Program Files\Microsoft\MRO\R-3.2.3\bin\x64\Rgui.exe" --quiet
Will launch the gui without the startup messages, same with R and RTerm.
But defining this seems to not pass that to the host before starting it:
![image](https://cloud.githubusercontent.com/assets/274303/15086700/7b39532c-13da-11e6-8b41-af47cab31801.png)

Maybe I defined something wrong?

**Comment by int19h** [comment]
No, it's a bug on our side - specifically, in our host. We call `R_set_command_line_arguments` which does propagate command line arguments to R, but then we never call `R_common_command_line`, which is where R actually parses them and sets options accordingly.

**Comment by int19h** [comment]
(so `commandArgs` work properly in R code and can be parsed by R scripts as usual, but all options affecting R behavior, like `--quiet`, are not applied.)

**Comment by Hlynsson** [comment]
Understood, could see people using things like --vanilla and --internet2 as well depending on environment conditions.
